draft-dns-zone-digest.xml

```diff
@@ -637,59 +610,99 @@ example.com. 86400 IN &RRNAME; 2018031500 1 0 (
           this step in the process, but it is harmless to do so.
         </t>
       </section>
-      <section title="Calculate the Digest" anchor="calculating-calculating">
+
+      <section title="Canonical Format and Ordering" anchor="canonical-ordering">
+        <t>
+          Calculation of a zone digest REQUIRES RRs
+          to be processed in a consistent format and ordering.  Correct ordering
+          depends on (1) ordering of owner names, (2)
+          ordering of RRSets with the same owner name, and (3) ordering of
+          RRs within an RRSet.
+        </t>
+        <t>
+          This specification adopts DNSSEC's canonical ordering for names
+          (Section 6.1 of <xref target="RFC4034"/>),
+          and canonical ordering for RRs within an RRSet
+          (Section 6.3 of <xref target="RFC4034"/>).
+          It also adopts DNSSEC's canonical RR form
+          (Section 6.2 of <xref target="RFC4034"/>).
+          However, since DNSSEC does not define a canonical ordering for RRSets
+          having the same owner name, that ordering is defined here.
+        </t>
+        <section title="Order of RRSets Having the Same Owner Name">
+          <t>
+            For the purposes of calculating the zone digest, RRSets having
+            the same owner name MUST be numerically ordered, in ascending order, by their numeric RR TYPE.
+          </t>
+        </section>
+      </section>
+
+      <!-- section title="Handling Duplicate RRs">
+        <t>
+          As stated in Section 5 of <xref target="RFC2181"/>
+          and Section 6.3 of <xref target="RFC4034"/>, it
+          is meaningless for a zone to have multiple RRs with
+          equal owner name, class, type, and RDATA.  In the
+          interest of consistency and interoperability,
+          only one instance of duplicate RRs with equal owner,
+          class, type and RDATA is included in a zone digest.
+        </t>
+      </section -->
```

Comment:
Why is this commented? (Question, not objection, although I do like it).

Reply:
I commented it out because after moving things around, this paragraph ended up right before the inclusion/exclusion rules and those rules repeat the sentence about handling duplicate RRs.  So it was redundant.  

By commenting this out, though, we lose the cites to RFC2181 and 4034.  Maybe those cites should be added to the inclusion/exclusion rule text.

Reply:
Any is fine with me.